packages/@react-aria/dialog/docs/useDialog.mdx

```diff
@@ -184,7 +184,7 @@ function Example() {
           isOpen
           onClose={state.close}
           isDismissable>
-          <form style={{display: 'flex', flexDirection: 'column'}}>
+          <form style={{display: 'flex', direction: 'column'}}>
```

Comment:
The submit button isn't aligned anymore:
![image](https://user-images.githubusercontent.com/9661127/84960456-82031d00-b0b6-11ea-98cc-a686366ed2de.png)